db/system_keyspace.cc

```
@@ -2198,6 +2198,49 @@ future<bool> system_keyspace::cdc_is_rewritten() {
     });
 }
 
+future<db_clock::time_point> system_keyspace::read_cdc_for_tablets_current_generation_timestamp(const sstring &ks_name, const sstring &table_name) {
+    static const sstring query = format("SELECT timestamp FROM {}.{} WHERE keyspace_name = ? and table_name = ? limit 1", NAME, CDC_TIMESTAMPS);
```

Is this a conscious decision to actually read only the 1st occurrence and silently ignore the others? What's the ordering?